src/inmanta/config.py

```diff
@@ -213,12 +213,14 @@ def is_set(cls, section: str, name: str) -> bool:
         return section in cls.get_instance() and name in cls.get_instance()[section]
 
     @classmethod
-    def getboolean(cls, section: str, name: str, default_value: Optional[bool] = None) -> bool:
+    def getboolean(cls, section: str, name: str, default_value: bool) -> bool:
         """
         Return a boolean from the configuration
         """
-        cls.validate_option_request(section, name, default_value)
-        return cls.get_instance().getboolean(section, name, fallback=default_value)
+        val = cls.get(section, name, default_value)
+        if val is None:
+            raise ValueError(f"Expected boolean value. Found: {val}")
```

Comment:
Not sure if `ValueError` is appropriate here. Afaik it refers to the argument values.

Reply:
I see. But this keeps the behavior consistent with the old behavior.